app/docs/0.10.x/secure-admin-api.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Securing the Admin API
+---
+
+# Overview
+
+Kong's Admin API provides a RESTful interface for administration and
+configuration of APIs, plugins, consumers, and credentials. Because this
+API allows full control of Kong, it is important to secure this API against
+unwanted access. This document touches on a few approaches to take to secure
```

Comment:
"...touches on a few approaches to take to secure..." can be shortened, and Colloquialism removed, by "...describes a few approaches to secure..."